packages/fx-core/src/component/utils/teamsFxTelemetryReporter.ts

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
This is a reporter which adds common properties to events before send. Expected to be shared in different components.

Reply:
@KennethBWSong Is it something duplicated design with the work you've done before?

Reply:
With some offline discussion, @KennethBWSong added some extra common properties previously but they will be removed in V3.